<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::TidyChecks</class>
 <widget class="QWidget" name="CppTools::TidyChecks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="androidCheck">
       <property name="text">
        <string>android-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="boostCheck">
       <property name="text">
        <string>boost-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="bugproneCheck">
       <property name="text">
        <string>bugprone-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="certCheck">
       <property name="text">
        <string>cert-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="coreCheck">
       <property name="text">
        <string>cppcoreguidelines-*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="analyzerCheck">
       <property name="text">
        <string>clang-analyzer-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="diagnosticCheck">
       <property name="text">
        <string>clang-diagnostic-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="googleCheck">
       <property name="text">
        <string>google-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="hicppCheck">
       <property name="text">
        <string>hicpp-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="llvmCheck">
       <property name="text">
        <string>llvm-*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="miscCheck">
       <property name="text">
        <string>misc-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="modernizeCheck">
       <property name="text">
        <string>modernize-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mpiCheck">
       <property name="text">
        <string>mpi-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="objcCheck">
       <property name="text">
        <string>objc-*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="performanceCheck">
       <property name="text">
        <string>performance-*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="readabilityCheck">
     <property name="text">
      <string>readability-*</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
